api/legacy_test.go

```diff
@@ -18,13 +19,13 @@ func TestLegacyHandler(t *testing.T) {
 		url := "/gdp/economy"
 		b := zebedee.Bulletin{URI: url, Description: zebedee.Description{Title: "Test"}}
 		mockZebedeeClient := &ZebedeeClientMock{
-			GetBulletinFunc: func(ctx context.Context, userAccessToken string, lang string, uri string) (zebedee.Bulletin, error) {
+			GetBulletinFunc: func(ctx context.Context, userAccessToken, collectionId, lang, uri string) (zebedee.Bulletin, error) {
 				return b, nil
 			},
 		}
 		handler := LegacyHandler(context.Background(), mockZebedeeClient)
 
-		Convey("when a valid request is received", func() {
+		Convey("when a valid request without headers is received (web)", func() {
 			req := httptest.NewRequest("GET", fmt.Sprintf("http://localhost:8080/articles/legacy?url=%s", url), nil)
```

Comment:
All good rafa, just if you wanted you could add the collection id:
req.Header.Set("Collection-Id", "One")
and then check its value

Reply:
Mmm not sure if I get you. This is specifically testing the case when there are no headers (as in web requests). There is another test case where I set them and check the value (publishing)